README.md

```diff
@@ -2,7 +2,18 @@
 Automated tests for Percona Monitoring and Management
 GUI tests are created for testing frontend of PMM. They include tests for Query Analytics and for Grafana dashboards
 
-## Installation (UI tests version 2.0)
+## Using Selenoid for running tests in Local
+1. Install Nodejs and atleast npm 8.x on your system
+2. Selenoid and Selenoid UI uses port 4444, 8080, make sure they are not being used, other change docker-compose.yml file.
```

Comment:
`Selenoid and Selenoid UI use port 4444 and 8080 respectively, make sure they are not being used, otherwise update docker-compose.yml file`